### PR TITLE
Real Time Text Component Related Changes and Strings 

### PR DESCRIPTION
**Pull request by carocao-msft**
# What
Real Time Text Component Related Changes and Strings 

# Why
New component to start real time text
Captions banner can now process real time text data 
Added new strings and icons 

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/communication-react/review/beta/communication-react.api.md [hunk 1]
```diff
@@ -5024,6 +5035,22 @@ export interface StartCaptionsButtonStrings {
     tooltipOnContent: string;
 }
 
+// @beta
+export const StartRealTimeTextButton: (props: StartRealTimeTextButtonProps) => JSX.Element;
+
+// @beta (undocumented)
```
Comment (dmceachernmsft) [inline packages/communication-react/review/beta/communication-react.api.md:5041]: if this is a public component lets add a comment here

File: packages/react-components/src/components/StartRealTimeTextButton.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+ */
+export interface StartRealTimeTextButtonStrings {
+  /**
+   * Label for when action is to start RealTimeText
+   */
+  onLabel: string;
+  /**
+   * Content for when button is checked, RealTimeText is on
+   */
+  tooltipOnContent: string;
```
Comment (dmceachernmsft) [inline packages/react-components/src/components/StartRealTimeTextButton.tsx:45]: is there any strings for Aria we need to add?

**Review by dmceachernmsft — APPROVED** [review]

File: packages/react-components/src/components/StartRealTimeTextButton.tsx (hunk 2 above)
Comment (carocao-msft) [reply packages/react-components/src/components/StartRealTimeTextButton.tsx:45]: No we don't need to. It's using the control bar buttons

File: packages/communication-react/review/beta/communication-react.api.md (hunk 1 above)
Comment (carocao-msft) [reply packages/communication-react/review/beta/communication-react.api.md]: Updated

File: packages/react-components/src/components/CaptionsBanner.tsx [hunk 3]
```diff
@@ -142,12 +182,37 @@ export const CaptionsBanner = (props: CaptionsBannerProps): JSX.Element => {
       scrollToBottom();
     }
   }, [captions, isAtBottomOfScroll]);
+  /* @conditional-compile-remove(rtt) */
+  const [textFieldValue, setTextFieldValue] = useState<string>('');
+  /* @conditional-compile-remove(rtt) */
+  useEffect(() => {
+    // if the latest real time text sent by myself is final, clear the text field
+    // find the last real time text caption
+    const lastRealTimeText = captions
```
Comment (alkwa-msft) [inline packages/react-components/src/components/CaptionsBanner.tsx:191]: do we think the selector could do this calculation for us instead?

**Review by alkwa-msft — APPROVED** [review]

File: packages/react-components/src/components/CaptionsBanner.tsx (hunk 3 above)
Comment (carocao-msft) [reply packages/react-components/src/components/CaptionsBanner.tsx]: Yep! Updated

**Pull request merged**